apps/ledger-live-mobile/e2e/bridge/start-server.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+/* 
+
+This script will allow you to upload app.json userdata files into LLM
+
+Steps:
+1. Run the react native app in one terminal: pnpm mobile start
+2. Get the name (excluding the .json file extension) of the app.json file you want in the e2e/setups folder. Run thes following in another terminal: pnpm mobile e2e:loadConfig <EXAMPLE FILE NAME>
+3. Run the mobile app in mock mode with: ENVFILE=.env.mock pnpm mobile ios
+4. Every time you reload the app or make a code change in LLM and save it, the app.json data will refresh
+
+*/
+
+import * as serverBridge from "./server";
+
+const filePath = process.argv[2];
+
+console.log("JSON file to look for: ", filePath);
```

Comment:
You might want to check if the filePath exist (the string and maybe the file itself too)